lib/generators/scenic/view/templates/db/migrate/update_view.erb

```diff
@@ -4,7 +4,7 @@ class <%= migration_class_name %> < <%= activerecord_migration_class %>
     update_view <%= formatted_plural_name %>,
       version: <%= version %>,
       revert_to_version: <%= previous_version %>,
-      materialized: true
+      materialized: true<%= ", materialized_no_data: true" if materialized_no_data? %>
```

Comment:
If we go this route:

```ruby
create_view :search_results, version: 1, materialized: { no_data: true }
```

Then we'd just set the materialized option to whatever the generator sets it to, which will either be a hash or false. We'd lose the discoverability bit, but I think that's ok.